.github/workflows/ci.yml

```diff
@@ -53,10 +55,6 @@ jobs:
           mkdir -p "$STACK_ROOT"
           echo "local-programs-path: $STACK_ROOT/programs" > $STACK_ROOT/config.yaml
 
-      - run: "ci/run-hlint.sh --git"
-        env:
-          VERSION: "2.2.11"
-
```

Comment:
I moved HLint into the same new job I created for Weeder because:
* They're both linty things
* I don't think there's much point to running HLint three times
* When all the caches are loaded up, this makes the longest paths shorter and so decreases the total time to passing all checks